src/static_ir/trace.jl

```diff
@@ -36,6 +36,10 @@ get_submap(choices::StaticIRTraceAssmt, addr::Pair) = _get_submap(choices, addr)
 
 abstract type StaticIRTrace <: Trace end
 
+function Base.getindex(trace::StaticIRTrace, addr)
+    get_choices(trace)[addr]
```

Comment:
```suggestion
    # There is no aux data, so `getindex` need only look at the choice map
    get_choices(trace)[addr]
```
right?

Reply:
Right. Adding aux data to static modeling language would be a separate PR.